### ISSUE TITLE
email tag

### ISSUE THREAD

**Issue by Alex-Jordan** [rbeezer/mathbook#600]
The `email` tag right now is only for use inside an `author`, `editor`, or `contributor` tag. AT least according to the DTD. But I think that it should be more versatile as markup, alongside `url`. Like in the middle of a`p`, you might say something like:
```
Send any typos you find to <email>name@place.ext</email>.
```

**Comment by rbeezer** [comment]
Agreed.  I'll be more careful about these sorts of things in the new schema (and so have tagged it as such).  In the meantime, it should just work as expected.

**Comment by rbeezer** [comment]
`email` is in the new schema as a purely visual thing - just a font change, and therefore it can be lots of places, like a `title`, and even inside the content of a `url`.

But it is implemented, both HTML and LaTeX, as an active `mailto:` link.  Do you think we need two forms, or perhaps a switch on this one (which is a bit delicate with the schema)?  Perhaps a discussion for the group?

**Comment by Jahrme** [comment]
We could have multiple forms of `email` depending both upon both the contents and the container.

For example, a `mailto` link can support more information than just the email address including `cc`, `bcc`, `subject`, and the `body` of an email (see the [w3schools.com](https://w3schools.com) example [here](https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_link_mailto2)).

As such, we should probably allow an email to be used in a simple fashion such as
```xml
<p>Please report errors by email at <email>email@example.com</email>.</p>
```
which could (in HTML) be converted to
```html
<p>Please report errors by email at <a class="email-url" href="mailto:email@example.com">email@example.com</a>.</p>
```

However, maybe we also allow authors to utilize some/all of the capabilities of a `mailto` link and utilize markup such as
```xml
<p>Please report any errors you find <email>
    <address>email@example.com</address>
    <cc>another-email@example.com</cc>
    <bcc>different-email@example.com</bcc>
    <subject>PTX Book Error</subject>
    <body>I found an error in [chapter, section, etc.] and it is [describe error].</body>
    <display>here</display>
</email>.</p>
```
which could (in HTML) be converted to
```html
<p>Please report any errors you find <a class="email-text" href="mailto:email@example.com?cc=another-email@example.com&bcc=different-email@example.com&subject=PTX%20Book%20Error&body=I%20found%20an%20error%20in%20[chapter,%20section,%20etc.]%20and%20it%20is%20[describe%20error]." target="_top">here</a>.</p>
```
where `address` is required, but `cc`, `bcc`, `subject`, `body` are optional (and maybe `display` as well).

---
The above HTML code (including a full `mailto` link) rendered by GitHub:
<p>Please report any errors you find <a class="email-text" href="mailto:email@example.com?cc=another-email@example.com&bcc=different-email@example.com&subject=PTX%20Book%20Error&body=I%20found%20an%20error%20in%20[chapter,%20section,%20etc.]%20and%20it%20is%20[describe%20error]." target="_top">here</a>.</p>

---
There may also be circumstances (e.g., if the `email` was part of a knowl clickable) were we want to turn off all `mailto` capabilities and convert `email` as purely visual markup so as to not nest links.

**Comment by rbeezer** [comment]
Thanks, Jahrme.  I hadn't realized you could add more like that.  That gives us 
more to chew on.

**Comment by rbeezer** [comment]
Implemented: `email` is now available at the text level (the schema's Verbatim group), so it can be used mid-sentence alongside `url` and `c`. Closing as delivered.

*Claude Fable 5, acting as a review assistant for Rob Beezer*
